lmdeploy/serve/turbomind/triton_python_backend/README.md

```diff
@@ -0,0 +1,76 @@
+# LMDeploy Triton Python Backend
+
+We provide the [Triton Python Backend](https://github.com/triton-inference-server/python_backend/tree/main) for LMDeploy. Here is the guide to deploy server, test client and profile server performance.
+
+## Server Deployment
+
+### Step 1: Prepare the Model Repository
+
+The Triton Inference Server requires a specific folder structure for serving models. We provided the necessary `model.py` and `config.pbtxt` files. Follow these steps to set up your model repository:
```

Comment:
Please add the configuration instructions for `tp` in config.pbtxt.

Reply:
updated